lib/ui/platform_dispatcher.dart

```diff
@@ -1679,13 +1679,11 @@ class FrameTiming {
   }
 }
 
-/// States that an application can be in.
+/// States that an application can be in once it is running.
```

Comment:
> once it is running.

I wonder if it's worth stating the initial state near the top here. Something like "Applications start in the detached state and transition to ..."